labs-ios-starter/Model Controller/UserController+API.swift

```diff
@@ -419,6 +419,64 @@ extension UserController {
         }
     }
 
+    func deleteTopic(with topicId: Int, completion: @escaping (Error?) -> Void) {
+        guard let oktaCredentials = getOktaAuth() else { return }
+
+        let requestURL = baseURL
+            .appendingPathComponent("topics")
+            .appendingPathComponent("topic")
+            .appendingPathComponent(topicId.description)
+
+        var request = URLRequest(url: requestURL)
+        request.httpMethod = "DELETE"
+        request.addValue("Bearer \(oktaCredentials.accessToken)", forHTTPHeaderField: "Authorization")
+
+        URLSession.shared.dataTask(with: request) { _, response, error in
```

Comment:
We should probably include the "isMock" and use the dataLoader here just to keep it consistent with all the other networking.

Reply:
I didn't include the isMock, because we seem to only have in "GET" requests

Reply:
I didn't include the isMock, because we seem to only have in "GET" requests